pkg/file/file_test.go

```diff
@@ -47,6 +77,64 @@ func TestLocalFileOpen(t *testing.T) {
 	}
 }
 
+func TestOpen_Combined(t *testing.T) {
+	openTestCases := []struct {
+		desc          string
```

Comment:
we have added desc for test cases, but we ain't using it anywhere. Should we update this?

Reply:
My bad. Will update the test method to use desc for logging in error case